app/workers/new_company_worker.rb

```diff
@@ -1,7 +1,11 @@
 class NewCompanyWorker
   include Sidekiq::Worker
 
-  def perform(id)
-    puts "Company #{id} created!"
+  def perform(company_id)
+    company = Company.find(company_id)
+    Page.create(page_type: Page::PAGE_TYPES.sample,
+                source_url: Faker::Internet.url('en.wikipedia.org/wiki'),
```

Comment:
why we have wikipedia here?

Reply:
you may also create page as `company.pages,create(<attributes_without_company>)`